internal/storage/types/ssblob/ssblob.go

```diff
@@ -0,0 +1,135 @@
+// Package ssblob provides a blob implementation for the Archivematica Storage
+// Service. Use OpenBucket to construct a *blob.Bucket.
+package ssblob
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+
+	"gocloud.dev/blob"
+	"gocloud.dev/blob/driver"
+	"gocloud.dev/gcerrors"
+)
+
+type NotImplemented error
+
+var errNotImpl NotImplemented
+
+type bucket struct {
+	Options
+}
+
+type Options struct {
+	Context context.Context
```

Comment:
I mentioned it before but to make sure this is addressed: we dont' need a context in the struct.